ert_gui/tools/plot/plot_api.py

```diff
@@ -208,20 +209,31 @@ def _add_index_range(self, data):
         index = pd.MultiIndex.from_tuples(tuples, names=["key_index", "data_index"])
         data.columns = index
 
-    def refcase_data(self, key):
-        """Returns a pandas DataFrame with the data points for the refcase for a
-        given data key, if any.  The row index is the index/date and the column
-        index is the key."""
-        return self._facade.refcase_data(key)
-
-    def history_data(self, key, case=None):
+    def history_data(self, key, case=None) -> pd.DataFrame:
         """Returns a pandas DataFrame with the data points for the history for a
         given data key, if any.  The row index is the index/date and the column
         index is the key."""
 
+        if str(key).endswith("H") or "H:" in str(key):
```

Comment:
Should there be an explanation on this  piece of code? From my knowledge i think i understand the reason you added it - that being the cases where we actually have keys ending with `H`  - it is not an historical vector? The "loose" concept here is quite fragile, and i support adding this check...

Reply:
Yes, if you select an historical vector this should be picked up here. I can add a comment for this line